xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-code/src/test/resources/macrocode16.test

```diff
@@ -2,6 +2,9 @@
 .#-----------------------------------------------------
 .input|xwiki/2.0
 .# Validates that the code macro accurately identifies the language used in the code macro when not specified.
+.# Pygments 2.4.2, and 2.0.8 identify the sample below as Tera Term macro, presenting an acceptable highlight of
```

Comment:
>  Tera Term macro

macro? You mean content?

Reply:
That's the human readable name of the TeraTermLexer.

Reply:
sure but your sentence doesn't make sense

Reply:
what you mean is "as Tera Term Macro content". You forgot "content". Same problem with "MIME" you cannot says it's recognized as MIME but as MIME content or syntax.